web_timeline/static/src/js/timeline_view.js

```diff
@@ -0,0 +1,186 @@
+/* global py */
+/* Odoo web_timeline
+ * Copyright 2015 ACSONE SA/NV
+ * Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define("web_timeline.TimelineView", function(require) {
+    "use strict";
+
+    const core = require("web.core");
+    const utils = require("web.utils");
+    const view_registry = require("web.view_registry");
+    const AbstractView = require("web.AbstractView");
+    const TimelineRenderer = require("web_timeline.TimelineRenderer");
+    const TimelineController = require("web_timeline.TimelineController");
+    const TimelineModel = require("web_timeline.TimelineModel");
+
+    const _lt = core._lt;
+
+    function isNullOrUndef(value) {
+        return _.isUndefined(value) || _.isNull(value);
+    }
+
+    var TimelineView = AbstractView.extend({
+        display_name: _lt("Timeline"),
+        icon: "fa-clock-o",
```

Comment:
¿Is this icon the same that in Actions used?
https://github.com/odoo/odoo/blob/67d2804d987ebd59a56f88218a8cec49e1a45db3/addons/mail/static/src/js/views/activity/activity_view.js#L16

Reply:
Yes, is the same icon... Any suggestion for a new one?

Reply:
Not sure which one uses OE gantt view, but what about this one?

https://fontawesome.com/icons/stream?style=solid

Reply:
Maybe https://fontawesome.com/v4.7.0/icon/tasks ?

Reply:
"stream" icon is available in font awesome version 5.0.13 ... so, i will use "tasks" icon that is available in 4.x ok?